src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+  }
+
+  static alive = [];
```

Comment:
The `Animal.alive` array is a static property, which means it is shared across all instances of `Animal`, `Herbivore`, and `Carnivore`. This might lead to unexpected behavior if you intended for each instance to have its own `alive` list. Consider whether this shared state is intentional.